kubectl_deploy/django/deployment.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: django
+  labels:
+    app: django
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: django-container
```

Comment:
The `container` bit here is a bit confusing I think - conceptually we're talking about a django app, not a container, so I'd call it `django` or `django-reference` or similar.

Reply:
Addressed under 9e8efe1.